HW12_1.py

```diff
@@ -0,0 +1,72 @@
+phone_book = {
+    'Igor': 132434,
+    'Diana': 254345,
+}
+
+import json
+
+json_data = json.dumps(phone_book)
+with open (r'C:\01_Education\03_IT\04_Pyhon\Laba\HW\phone_book.txt', 'a') as file:
```

Comment:
It is better to use a relative path to the file here (just open('phone_book.txt`) since the current path is valid only for your computer. And for json data you can use .json extension

Reply:
seems to have done